fdbclient/Schemas.cpp

```diff
@@ -482,6 +482,8 @@ const KeyRef JSONSchemas::statusSchema = LiteralStringRef(R"statusSchema(
 )statusSchema"
                                                           R"statusSchema(
       "recovery_state":{
+         "time_since_last_db_turned_available_seconds":1,
```

Comment:
```suggestion
         "time_since_last_recovered":1,
```

Reply:
Changes here also need to go in documentation/sphinx/source/mr-status-json-schemas.rst.inc